.travis-ci.sh

```diff
@@ -72,11 +72,11 @@ EOF
   export PATH=$PREFIX/bin:$PATH
   $MAKE world.opt
   $MAKE ocamlnat
-  cd testsuite
+  cd testsuite || exit 1
   echo Running the testsuite with the normal runtime
   $MAKE all
   echo Running the testsuite with the debug runtime
-  $MAKE USE_RUNTIME="d" OCAMLTESTDIR=$(pwd)/_ocamltestd TESTLOG=_logd all
+  $MAKE USE_RUNTIME="d" OCAMLTESTDIR="$(pwd)/_ocamltestd" TESTLOG=_logd all
```

Comment:
I suppose, although `$(pwd)` in a Travis setup is highly unlikely to start containing spaces suddenly.